pkgs/development/python-modules/enzyme/default.nix

```diff
@@ -2,24 +2,27 @@
   lib,
   fetchPypi,
   buildPythonPackage,
+  setuptools,
 }:
 
 buildPythonPackage rec {
   pname = "enzyme";
-  version = "0.4.1";
-  format = "setuptools";
+  version = "0.5.2";
+  pyproject = true;
 
   # Tests rely on files obtained over the network
   doCheck = false;
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "1fv2kh2v4lwj0hhrhj9pib1pdjh01yr4xgyljhx11l94gjlpy5pj";
+    hash = "sha256-fPd5FI2eZusoOGA+rOFAxTw878i4/l1NWgOl+11Xs8E=";
   };
 
+  propagatedBuildInputs = [ setuptools ];
```

Comment:
Are you sure this shouldn't be `nativeBuildInputs`?

Reply:
Ah oops, fixed